packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -2306,32 +2280,6 @@ describe('src/cy/commands/navigation', () => {
         return null
       })
     })
-
-    it('does not wait for stability at the end of the command queue when not stable with experimentalSessionAndOrigin', (done) => {
```

Comment:
test moved to the stability test file

Reply:
Now it's been combined with the test here, so it flexes the test based on the experimental flag